src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -110,33 +93,56 @@ void updateUseSplitter(boolean force) {
 
     trackRebuildsCheckbox.addChangeListener((l) -> {
       setTrackRebuildWidgets(trackRebuildsCheckbox.isSelected());
-      updateUseSplitter(false);
+      updateShowPerfSummaryView();
     });
 
     if (ENABLE_TRACK_REPAINTS) {
       trackRepaintsCheckbox.addChangeListener((l) -> {
         setTrackRepaintWidgets(trackRepaintsCheckbox.isSelected());
-        updateUseSplitter(false);
+        updateShowPerfSummaryView();
       });
     }
   }
 
+  void updateShowPerfSummaryView() {
+    final boolean show = getShowPerfTable();
+    final boolean firstRender = currentSummaryView == null;
+    final JComponent summaryView = show ? perfSummaryView : perfSummaryPlaceholder;
+
+    if (summaryView != currentSummaryView) {
+      if (currentSummaryView != null) {
+        perfSummaryContainer.remove(currentSummaryView);
+      }
+      currentSummaryView = summaryView;
+      perfSummaryContainer.add(summaryView, BorderLayout.CENTER);
+      perfSummaryContainer.revalidate();
+      perfSummaryContainer.repaint();
+    }
+  }
+
+  boolean getShowPerfTable() {
+    final FlutterWidgetPerfManager widgetPerfManager = FlutterWidgetPerfManager.getInstance(app.getProject());
+    return widgetPerfManager.isTrackRebuildWidgets() || widgetPerfManager.isTrackRebuildWidgets();
+  }
+
   private void buildUI() {
     setLayout(new BorderLayout(0, 8));
     setBorder(JBUI.Borders.empty(3));
 
     // Header
+    final JPanel footer = new JPanel(new VerticalLayout(0));
+    footer.add(new JSeparator());
     final JPanel labels = new JPanel(new BorderLayout(6, 0));
     labels.setBorder(JBUI.Borders.empty(0, 8));
-    add(labels, BorderLayout.NORTH);
 
     labels.add(
       new JBLabel("Running in " + app.getLaunchMode() + " mode"),
```

Comment:
Is there a way to specify that the label contents should align to the top vertically?

Reply:
Good point. Done.
<img width="1069" alt="screen shot 2018-11-27 at 8 17 28 am" src="https://user-images.githubusercontent.com/1226812/49095258-f3515580-f21c-11e8-904c-3bd53b8fb91f.png">